cfme/tests/cloud_infra_common/test_vm_instance_analysis.py

```diff
@@ -371,6 +404,83 @@ def test_ssa_template(local_setup_provider, provider, soft_assert, vm_analysis_p
 
 
 @pytest.mark.rhv3
+@pytest.mark.tier(2)
+@pytest.mark.long_running
+def test_ssa_compliance(local_setup_provider, ssa_compliance_profile, ssa_profiled_vm, ssa_vm,
+                        soft_assert, appliance):
+    """ Tests SSA can be performed and returns sane results
+
+    Metadata:
+        test_flag: vm_analysis
+    """
+    e_users = None
+    e_groups = None
+    e_packages = None
+    e_services = None
+    e_os_type = ssa_vm.system_type['os_type']
+
+    if ssa_vm.system_type != WINDOWS:
+        e_users = ssa_vm.ssh.run_command("cat /etc/passwd | wc -l").output.strip('\n')
+        e_groups = ssa_vm.ssh.run_command("cat /etc/group | wc -l").output.strip('\n')
+        e_packages = ssa_vm.ssh.run_command(
+            ssa_vm.system_type['package-number']).output.strip('\n')
+        e_services = ssa_vm.ssh.run_command(
+            ssa_vm.system_type['services-number']).output.strip('\n')
+
+    logger.info("Expecting to have %s users, %s groups, %s packages and %s services", e_users,
+                e_groups, e_packages, e_services)
+    view = appliance.browser.create_view(TasksView)
+    wait_for(
```

Comment:
Required: Please add message for wait for 
ex. message="Waiting for SSA runs for {} vm".format(ssa_vm.name)

Reply:
Okay will add it.